.schemas/config.schema.json

```diff
@@ -678,6 +678,35 @@
               }
             }
           ]
+        },
+        "retry": {
+          "type": "object",
+          "properties": {
+            "conn_timeout": {
```

Comment:
I like `connection_timeout` more but we are already using `max_delay` in another retry definition so I would like to use `max_delay` here as well: https://github.com/ory/oathkeeper/blob/2c310f95063233b2d48d105a476f674516336537/.schemas/config.schema.json#L827-L836